### ISSUE TITLE
ServerClientId needed for GoogleSignInOptions Web server OAuth

### ISSUE THREAD

**Issue by ersin-ertan** [googlesamples/easygoogle#24]
TL;DR The issue is with not having the ability to include a serverClientId in the GoogleSignInOptions.

This problem appeared when [GoogleSignInAccount.getIdToken();](https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInAccount.html#public-methods) (which is required for my backend's google plus login) was returning null.

Sign in was a clean abstraction and was great to work with, however, I believe the use case with [registering OAuth clients for Google Sign-In](http://android-developers.blogspot.sg/2016/03/registering-oauth-clients-for-google.html) states that "Web server OAuth client registration" is needed with android by requesting/ "an ID token in your GoogleSignInOptions, supplying the web client ID for your server". 
```java
GoogleSignInOptions gso =
    new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
        .requestIdToken(serverClientId)
        .requestEmail()
        .build();
```

EasyGoogle does not have us directly implement the GoogleSignInOptions class. From what I understand, the [IdTokenActivity](https://github.com/googlesamples/google-services/blob/master/android/signin/app/src/main/java/com/google/samples/quickstart/signin/IdTokenActivity.java) is the original solution, which voids the use of easygoogle. See line [#55.](https://github.com/googlesamples/google-services/blob/master/android/signin/app/src/main/java/com/google/samples/quickstart/signin/IdTokenActivity.java#L55)

**Comment by samtstern** [comment]
@ersin-ertan this is a good feature request.  It would be as simple as adding an optional parameter to the [`enableSignIn`](https://github.com/googlesamples/easygoogle/blob/master/easygoogle/src/main/java/pub/devrel/easygoogle/Google.java#L69) method to take serverClientId, and then passing it through to GoogleSignInOptions when it's non-null.

Would you consider submitting a pull request for this functionality?  I'd be happy to review it.

**Comment by ersin-ertan** [comment]
See merged pull request https://github.com/googlesamples/easygoogle/pull/25
